ui/mirage/handlers/clients.js

```diff
@@ -1,4 +1,4 @@
-import { formatISO, isAfter, isBefore, sub, isSameMonth, startOfMonth } from 'date-fns';
+import { addDays, formatISO, formatRFC3339, isAfter, isBefore, sub, isSameMonth, startOfMonth } from 'date-fns';
 import { parseAPITimestamp } from 'core/utils/date-formatters';
 const MOCK_MONTHLY_DATA = [
```

Comment:
Ah, so this mirage file is actually not used for running tests at the moment. (That's the end goal, but right now it's just used for local development)

Reply:
Ah, I my recollection seems to be incorrect and that I modified the wrong `sys/license/status` mock. Whoops, thanks for calling that out!